test/models/database/user_repository_test.dart

```diff
@@ -0,0 +1,162 @@
+import "package:cloud_firestore/cloud_firestore.dart";
+import "package:fake_cloud_firestore/fake_cloud_firestore.dart";
+import "package:firebase_auth_mocks/firebase_auth_mocks.dart";
+import "package:flutter_test/flutter_test.dart";
+import "package:proxima/models/database/user_repository.dart";
+
+void main() {
+  group("User repository testing", () {
```

Comment:
You don't have a test for hashing in this one, as you have for the `PostRepository` tests. I don't think this is strictly necessary to implement (I'm not sure I would have added a test for hashes in the first place, even though more testing is better than less testing), but I just wanted to point it out.

Reply:
Yes, I think I forgot, I'll add it for consistency. 
Fix in 57d2ca7a0df7e3f6b09b9568cde48d7c289e6b66.